crates/tinymist/src/task/format.rs

```diff
@@ -39,8 +40,15 @@ impl FormatTask {
             match c.mode {
                 FormatterMode::Typstyle => {
                     let cw = c.width as usize;
-                    let res = typstyle_core::Typstyle::new_with_src(src.clone(), cw).pretty_print();
-                    Ok(calc_diff(src, res, c.position_encoding))
+                    let res = typstyle_core::Typstyle::new_with_src(
+                        src.clone(),
+                        PrinterConfig::new_with_width(cw),
+                    )
+                    .pretty_print();
+                    match res {
+                        Ok(res) => Ok(calc_diff(src, res, c.position_encoding)),
+                        Err(_) => Ok(None),
```

Comment:
currently the only err is syntax error. pls check if this makes sense to you

Reply:
It looks ok to me, but I converted the `match` to a single `.ok()` to ignore the error.